### PR TITLE
Fix locales for Date

### PR DESCRIPTION
**Pull request by Magnus93**

### CODE REVIEW

File: Handler/date/helper.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import * as isoly from "isoly"
+import { DateFormat } from "../../DateFormat"
+import { State } from "../../State"
+
+export function validDay(symbol: string, state: Readonly<State>, startIndex: number, endIndex: number): boolean {
```
Comment (nilssonemma) [inline Handler/date/helper.ts:5]: Please only have 1 return statement per function. Use a result variable instead

**Review by robinnedergard — DISMISSED** [review]

File: Handler/date/helper.ts (hunk 1 above)
Comment (Magnus93) [reply Handler/date/helper.ts:5]: Fixed

**Review by nilssonemma — DISMISSED** [review]

File: Handler/date/index.ts [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+		case "mm/dd/YYYY":
+			year = parseInt(date.substring(6, 10))
+			month = parseInt(date.substring(0, 2))
+			day = parseInt(date.substring(3, 5))
+			return year && month && day ? day > 0 && daysPerMonth(year, month) >= day : false
+		default:
+			year = parseInt(date.substring(0, 4))
+			month = parseInt(date.substring(5, 7))
+			day = parseInt(date.substring(8, 10))
+			return year && month && day ? day > 0 && daysPerMonth(year, month) >= day : false
```
Comment (robinnedergard) [inline Handler/date/index.ts]: one return per function

**Review by robinnedergard — CHANGES_REQUESTED** [review]

File: Handler/date/index.ts (hunk 2 above)
Comment (Magnus93) [reply Handler/date/index.ts]: Fixed

File: Handler/date/index.ts [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+							: validDate(result.value.substring(0, 8) + "28")
+							? result.value.substring(0, 8) + "28"
+							: result.value
+					)
+				break
+			}
+	}
+	return result
+
+	return unformated
```
Comment (robinnedergard) [inline Handler/date/index.ts]: Like this?:
```suggestion
	return result
```

**Review by robinnedergard — CHANGES_REQUESTED** [review]

**Review by robinnedergard — APPROVED** [review]

**Review by nilssonemma — APPROVED** [review]

File: Handler/date/index.ts (hunk 3 above)
Comment (Magnus93) [reply Handler/date/index.ts]: Fixed

**Review by fredrikbryntesson — APPROVED** [review]

**Pull request merged**